tree/master/packages/extension/public/locales/pl/translation.json

```diff
@@ -0,0 +1,162 @@
+	  "copy address": "kopiuj adres",
+	  "account visibility": "widoczność konta",
+	  "Wrong password": "Złe hasło",
+	  "Incorrect derivation path": "Nieprawidłowa ścieżka wyprowadzenia",
+	  "lifetime": "dożywotni",
+	  "Warning: Caps lock is on": "Ostrzeżenie: Caps lock jest włączony",
+	  "Invalid Json file": "Nieprawidłowy plik JSON",
+	  "Unable to decode using the supplied passphrase": "Nie można zdekodować przy użyciu dostarczonego hasła",
+	  "Manage Website Access": "Zarządzaj dostępem do witryny",
+	  "example.com": "example.com",
+	  "No website request yet!": "Nie ma jeszcze żądania strony!",
+	  "allowed": "dozwolony",
+	  "denied": "odmówiono",
+	  "Derive from an account": "Wyprowadź z konta",
+	  "Dark": "Ciemny",
+	  "Light": "Jasny",
+	  "Invalid mnemonic seed or derivation path": "Nieprawidłowy inicjator mnemoniczny lub ścieżka derywacyjna",
+	  "Invalid mnemonic seed": "Nieprawidłowe ziarno mnemoniczne",
+	  "Mnemonic needs to contain 12, 15, 18, 21, 24 words": "Mnemonik musi zawierać 12, 15, 18, 21, 24 słowa",
+	  "advanced": "zaawansowane",
+	  "derivation path": "ścieżka wyprowadzenia",
+	  "Next": "Następny",
+	  "//hard": "//ciężko ",
+	  "`///password` not supported for derivation": "`///password` nie jest obsługiwane przy wyprowadzaniu",
+	  "Soft derivation is only allowed for sr25519 accounts": "Miękkie wyprowadzanie jest dozwolone tylko dla kont sr25519",
+	  "Invalid derivation path": "Nieprawidłowa ścieżka wyprowadzenia",
+	  "hardware wallet account": "konto portfela sprzętowego",
+	  "Camera access must be first enabled in the settings": "Dostęp do kamery musi być najpierw włączony w ustawieniach",
+	  "Ledger devices can only be connected with Chrome browser": "Urządzenia Ledger mogą być połączone tylko z przeglądarką Chrome",
+	  "Attach ledger account": "Załącz konto Ledger",
+	  "Connect Ledger device": "Podłącz urządzenie Ledger",
+	  "External accounts and Access": "Konta zewnętrzne i dostęp",
+	  "Allow QR Camera Access": "Zezwól na dostęp do kamery QR",
+	  "Add Account": "Dodaj konto",
+	  "Account type {{index}}": "Typ konta {{index}}",
+	  "Address index {{index}}": "Indeks adresu {{index}}",
+	  "Import Ledger Account": "Importuj Konto Ledger",
+	  "Network": "Sieć",
+	  "account type": "typ konta",
+	  "address index": "indeks adresu",
+	  "Refresh": "Odśwież",
+	  "Import Account": "Importuj konto",
+	  "Sign on Ledger": "Podpisz się w Ledger",
+	  "Allow use on any chain": "Zezwól na użycie w dowolnym łańcuchu",
+	  "Select network": "Wybierz sieć",
+	  "You have been redirected because the Polkadot{.js} extension believes that this website could compromise the security of your accounts and your tokens.": "Zostałeś przekierowany ponieważ wtyczka Polkadot{.js} uważa że ta strona może naruszyć bezpieczeństwo swoich kont i swoimi żetonami}.",
```

Comment:
```suggestion
	  "You have been redirected because the Polkadot{.js} extension believes that this website could compromise the security of your accounts and your tokens.": "Zostałeś przekierowany ponieważ wtyczka Polkadot{.js} uważa że ta strona może naruszyć bezpieczeństwo Twoich kont i aktywów}.",
```